packages/IMAPClient-UI.package/ICLoginDialog.class/instance/listKey.from..st

```diff
@@ -0,0 +1,7 @@
+initialization
+listKey: a from: b
+	Transcript show: a.		
+			
```

Comment:
Here are a lot of empty spaces too :3.

Reply:
Thats a debug method, shouldnt be here^^